garak/probes/audio.py

```diff
@@ -0,0 +1,104 @@
+# SPDX-FileCopyrightText: Portions Copyright (c) 2023 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+"""Audio attack probes.
+
+Probes designed to test audio-to-text models and the audio component of multimodal models.
+"""
+
+import logging
+from typing import Iterable
+
+import garak.attempt
+from garak import _config
+from garak.probes.base import Probe, Tier
+from garak.exception import GarakException
+
+
+class AudioAchillesHeel(Probe):
```

Comment:
Consistency change from `main`:
```suggestion
class AudioAchillesHeel(garak.probes.Probe):
```